### ISSUE TITLE
3 phBB Debugs in Explorer

### ISSUE THREAD

**Issue by thaumatology** [RolePlayGateway/roleplaygateway#312]
- entered explorer
- assets dont load for location data
- error codes in black at top of screen, 3 phpbb debugs

```
[phpBB Debug] PHP Notice: in file /explorer/lib/class/RPGChat.php on line 151: session_start(): open(/tmp/php/session/sess_scm2qf1j58071q91lg9er6s3b1, O_RDWR) failed: No such file or directory (2)

[phpBB Debug] PHP Notice: in file Unknown on line 0: Unknown: open(/tmp/php/session/sess_g7ep6p48j0njopg13lpb2jk3j4, O_RDWR) failed: No such file or directory (2)

[phpBB Debug] PHP Notice: in file Unknown on line 0: Unknown: Failed to write session data (files). Please verify that the current setting of session.save_path is correct (/tmp/php/session)
```

<img width="1453" alt="Screen Shot 2022-09-26 at 3 34 07 PM" src="https://user-images.githubusercontent.com/31420084/192366126-9794447a-46b7-4567-bae7-9b2b8aa9c898.png">
